client/src/app/app-routing.module.ts

```diff
@@ -29,6 +30,7 @@ const routes: Routes = [
   { path: "industry-dashboard", component: IndustryDashboardComponent, canActivate: [AuthGuard] },
   //This route has two URL parameters, one for projectID, and one for applicationID. It is used to access a specific applicant's data by the faculty
   { path: "application/:projectID/:applicationID", component: ViewApplicationComponent },
+  { path: "student-search-opps", component: StudentSearchOppsComponent, canActivate: [AuthGuard] },
```

Comment:
Can you resolve the merge conflicts without removing any other routes on the front end?